package.json

```diff
@@ -71,6 +71,9 @@
     "react-transform-hmr": "^1.0.1",
     "redbox-react": "^1.0.5",
     "redux": "^3.0.0",
+    "redux-devtools": "^3.0.0-beta-3",
```

Comment:
Then we have to wait til release. I am trying to have Este as stable as possible.